### ISSUE TITLE
Error: Accessibility enabled: fail while trying to run the app

### ISSUE THREAD

**Issue by mendieta** [withfig/fig#600]
### Description:
> Please include a detailed description of the issue (and an image or screen recording, if applicable)

### Details:
|macOS|Fig|Shell|
|-|-|-|
|11.4.20F71|1.0.52|/usr/local/Cellar/zsh/5.8_1/bin/zsh|
<details><summary><code>fig diagnostic</code></summary>
<p>
<pre>Fig Version: Version 1.0.52 (B313) [ABC] 
UserShell: /usr/local/bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/mendieta/.fig/bin/fig
Accessibility: false
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path: <none>
iTerm Integration: installed!
Hyper Integration: application is not present.
VSCode Integration: installed!
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: 
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: ??? (???) - ???
Current working directory: /Users/mendieta
Current window identifier: ???
Path: /Users/mendieta/.volta/bin:/usr/local/opt/mysql@5.7/bin:/Users/mendieta/go/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/mendieta/.cargo/bin:/Users/mendieta/.fig/bin:/Users/mendieta/Library/Caches/antibody/https-COLON--SLASH--SLASH-github.com-SLASH-unixorn-SLASH-git-extra-commands/bin:/Users/mendieta/Library/Android/sdk/emulator:/Users/mendieta/Library/Android/sdk/tools:/Users/mendieta/Library/Android/sdk/tools/bin:/Users/mendieta/Library/Android/sdk/platform-tools:/Users/mendieta/SDKs/flutter/bin:/Users/mendieta/.pub-cache/bin</pre>
</p>
</details>

**Comment by mschrage** [comment]
Hey @mendieta! So sorry you are running into issues.

 Can you click this button in the menu bar?
<img width="497" alt="Screen Shot 2021-11-17 at 12 18 25 PM" src="https://user-images.githubusercontent.com/3155109/142275958-3bc122ef-c437-40c9-98cb-d702b9b88610.png">

This should open up System Preferences to the Accessibility page. You will then need to toggle the checkmark next to the Fig icon.

<img width="780" alt="Screen Shot 2021-11-17 at 12 20 23 PM" src="https://user-images.githubusercontent.com/3155109/142276186-8b269c18-547e-48a4-b067-fc7ffe04ccc0.png">

Let me know if you're still running into issues.
